kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -42,6 +42,8 @@
 _LOGGER: Final = logging.getLogger(__name__)
 _LOG_FORMAT: Final = '%(levelname)s %(asctime)s %(name)s - %(message)s'
 
+sys.setrecursionlimit(10**8)
```

Comment:
This should be set in `main`, i.e. replace the value there with the new, higher value.